tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/MockTest.java

```diff
@@ -167,6 +167,19 @@ public void testObjectMocking_NegativeCases(String message, String test, String
         AssertionUtils.assertOutput(baseOutputFile, output);
     }
 
+    @Test
+    public void testObjectMocking_NonPublicField() throws BallerinaTestException, IOException {
+        String[] args = mergeCoverageArgs(new String[]{"--tests",
+                "non_public_field_mock:testNonPublicMemberFieldMock"});
+        String output =
+                balClient.runMainAndReadStdOut("test", args, new HashMap<>(),
+                        projectBasedTestsPath.resolve("non-public-field-mock").toString(), false);
+        String firstString = "Generating Test Report\n\t";
```

Comment:
Having `\n\t` causes issues in actual ground truth outputs in windows. A corrupted output is generated. I have added a fix here(https://github.com/ballerina-platform/ballerina-lang/pull/40241). Can we wait until that PR is merged.